wikiclue/src/routes/manage-friends/+page.svelte

```diff
@@ -0,0 +1,41 @@
+<script lang="ts">
+    import HeaderBar from "../../components/headerBar.svelte";
+    import Overlay from '../../components/overlay.svelte';
+    import { writable } from 'svelte/store';
+
+    const friendRemoveOverlayVis = writable(false);
+
+    function removeFriend(){
+        //remove friend logic
+    }
+</script>
+
+<HeaderBar />
+<div class="manage-friends">
+    <h1>Manage Friends</h1>
+    <div class="friendboard-container">
+        <div class="friends-container">
+            <h3>View and manage your friends!</h3>
+            <div class ="users-container">
```

Comment:
Can we also put this in a scrollview in case there are more users than can fit in the box?